styles/style.css

```diff
@@ -158,11 +191,18 @@ p {
   background: var(--orange-dark);
 }
 
-
 /* Article */
 
+section {
+  display: flex;
+}
+
 .article {
+  display: flex;
```

Comment:
Same comment as for Ezgi -- you set the `display: flex` on the _parent_ container. So to make articles behave like flex items, you want this setting on the parent of the `.article` class -- unless you intended to make the text and headings within `.article` behave like flex items?

You can read more about the parent (container)/child (item) properties here: https://css-tricks.com/snippets/css/a-guide-to-flexbox/